deploy/operator.yaml

```diff
@@ -44,16 +44,16 @@ spec:
             - name: ZYNC_IMAGE
               value: "quay.io/3scale/zync:nightly"
             - name: SYSTEM_MEMCACHED_IMAGE
-              value: "memcached:1.5"
+              value: "registry.redhat.io/3scale-amp2/memcached-rhel7:3scale2.8"
```

Comment:
Should we use 3scale2.9 here?

Reply:
the image needs to exist. When 2.9 goes to GA, we have to update all links, like done for 2.8

Reply:
:100: